frontend/components/Dashboard/PostCard/PostCard.tsx

```diff
@@ -89,7 +90,8 @@ const PostCard: React.FC<Props> = ({
                 )}
 
                 <div className="post-subtext">
-                  {formatShortDate(createdAt)} - {t('readTime', { minutes: readTime })}
+                  {publishedAt ? formatShortDate(publishedAt) : formatShortDate(createdAt)} -{' '}
```

Comment:
```suggestion
                  {formatShortDate(publishedAt || createdAt) -{' '}
```

Reply:
sorry, missing curly brace after closing paren

Reply:
Got it ✅